packages/xo-web/src/common/select-objects.js

```diff
@@ -229,7 +229,12 @@ class GenericSelect extends React.Component {
           : undefined
       }
     >
-      {renderXoItem(option.xoItem)}
+      {renderXoItem(option.xoItem, {
+        type:
+          this.props.resourceSet &&
+          option.xoItem.type === 'SR' &&
+          `${option.xoItem.type}-resourceSet`,
```

Comment:
So `type` may be `undefined`?

Reply:
yes if it's `undefined`, the type will be item's type.

Reply:
This would work because [the test is not explicit](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/render-xo-item.js#L308) but please make it an explicit `undefined` here.